test/functional/aws_standalone_vpc.go

```diff
@@ -0,0 +1,582 @@
+		Namespace: awsCredsNamespace,
+		Name:      strategyMapName,
+	}, strategyMap)
+	if err != nil {
+		t.Fatal("could not get aws strategy map", err)
+	}
+
+	// get the create strategy for _network in the aws strategy configmap
+	// if this doesn't exist, skip the test completely since we're dealing
+	// with legacy cro networking
+	strat, err := getStrategyForResource(strategyMap, resourceType, tier)
+	if err != nil {
+		t.Skip("_network key does not exist in aws strategy configmap, skipping standalone vpc network test")
+	}
+
+	// get the vpc cidr block
+	expectedCidr, err := getCidrBlock(strat)
+	if err != nil {
+		t.Fatal("could not get cidr block", err)
+	}
+
+	// get the cluster id used for tagging aws resources
+	clusterTag, err := getClusterID(ctx, testingCtx.Client)
+	if err != nil {
+		t.Fatal("could not get cluster id", err)
+	}
+
+	// verify vpc
+	vpc, err := verifyVpc(ec2Sess, clusterTag, expectedCidr)
+	testErrors.vpcError = err.(*networkConfigTestError).vpcError
+
+	// fail immediately if the vpc is nil, since all of the
+	// other network components are associated with it
+	if vpc == nil {
+		t.Fatal(testErrors.Error())
+	}
+
+	// verify subnets
+	subnets, err := verifySubnets(ec2Sess, clusterTag, expectedCidr)
+	testErrors.subnetsError = err.(*networkConfigTestError).subnetsError
+
+	// verify security groups
+	err = verifySecurityGroup(ec2Sess, clusterTag)
+	testErrors.securityGroupError = err.(*networkConfigTestError).securityGroupError
+
+	// we have to manually construct the subnet group names for rds and elasticache,
+	// since tag filtering isnt currently available
+	name := resources.ShortenString(fmt.Sprintf("%s-%s", clusterTag, "subnet-group"), 40)
+
+	// build array list of all vpc private subnets
+	var subnetIDs []*string
+	for _, subnet := range subnets {
+		subnetIDs = append(subnetIDs, subnet.SubnetId)
+	}
+
+	// verify rds subnet groups
+	rdsSvc := rds.New(session)
+	err = verifyRdsSubnetGroups(rdsSvc, name, subnetIDs)
+	testErrors.rdsSubnetGroupsError = err.(*networkConfigTestError).rdsSubnetGroupsError
+
+	// verify elasticache subnet groups
+	cacheSvc := elasticache.New(session)
+	err = verifyCacheSubnetGroups(cacheSvc, name, subnetIDs)
+	testErrors.cacheSubnetGroupsError = err.(*networkConfigTestError).cacheSubnetGroupsError
+
+	// verify peering connection
+	conn, err := verifyPeeringConnection(ec2Sess, clusterTag, expectedCidr, aws.StringValue(vpc.VpcId))
+	testErrors.peeringConnError = err.(*networkConfigTestError).peeringConnError
+
+	// verify standalone vpc route table
+	err = verifyStandaloneRouteTable(ec2Sess, clusterTag, conn)
+	testErrors.standaloneRouteTableError = err.(*networkConfigTestError).standaloneRouteTableError
+
+	// verify cluster route table
+	err = verifyClusterRouteTables(ec2Sess, clusterTag, expectedCidr, conn)
+	testErrors.clusterRouteTablesError = err.(*networkConfigTestError).clusterRouteTablesError
+
+	// if any error was found, fail the test
+	if testErrors.isValid() {
```

Comment:
nit: `isValid` is a potentially confusing in the context of running tests where we want to not have errors. Maybe `hasErrors` or something similar would be a bit more obvious?

Reply:
Sure, let me change this :+1: